src/online2/online-nnet3-decoding.h

```diff
@@ -0,0 +1,132 @@
+// online2/online-nnet3-decoding.h
+
+// Copyright 2014  Johns Hopkins University (author: Daniel Povey)
```

Comment:
online-nnet3-decoding.h is in online2 because online-nnet2-decoding.h is online2 as well.

If I move this, then nnet3 will depend on online2, is this ok?